site/zenodo_rdm/assets/semantic-ui/js/zenodo_rdm/src/support/components/SupportForm.js

```diff
@@ -0,0 +1,290 @@
+import { PropTypes } from "prop-types";
+import React, { Component } from "react";
+import { Formik, Form } from "formik";
+import { Form as SemanticForm, Button, Modal } from "semantic-ui-react";
+import { TextField, TextAreaField, ToggleField } from "react-invenio-forms";
+import CategoryDropdown from "./CategoryDropdown";
+import * as Yup from "yup";
+import axios from "axios";
+import Dropzone from "react-dropzone";
+import FileTable from "./FileTable";
+import _remove from "lodash/remove";
+import { humanReadableBytes } from "react-invenio-deposit";
+
+// TODO translations
+// TODO implement error notification
+// TODO implement recaptcha
+// TODO required fields are highlighted by the component and not Formik
+
+const apiConfig = {
+    withCredentials: true,
+    xsrfCookieName: "csrftoken",
+    xsrfHeaderName: "X-CSRFToken",
+    headers: {
+        "Accept": "application/json",
+        'Content-Type': 'multipart/form-data',
+    },
+};
+
+const http = axios.create(apiConfig);
+
+function formikToFormData(values) {
```

Comment:
Even though `Formik` stores files, they were not being sent properly to the backend. Thus, when posting the form data I had to use `FormData`. 

If there is a cleaner / more correct way to do it, please let me know.